packages/aws-cdk/lib/init-templates/app/python/hello/hello_stack.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+from aws_cdk import (
+    aws_iam as iam,
+    aws_sqs as sqs,
+    aws_sns as sns,
+    cdk
+)
+
+from hello_construct import HelloConstruct
+
+class MyStack(cdk.Stack):
+
+    def __init__(self, app: cdk.App, id: str) -> None:
+        super().__init__(app, id)
+
+        queue = sqs.Queue(
```

Comment:
I am wondering if perhaps it would be nicer if we settle on a style where the first two arguments are in the same line as the ctor and then the kwargs are in separate lines:

```py
queue = sqs.Queue(self, "MyFirstQueue",
                                  visibility_timeout_sec=300)
```

I don't know if that's pythonic or not, but may work better and a bit more concise for the CDK idioms that you always have the two arguments.

Reply:
There is a similar style used in boto3 examples, https://boto3.amazonaws.com/v1/documentation/api/latest/guide/cw-example-creating-alarms.html#id3.

Reply:
So, we could adopt that here.  I have no objections.